frontend/src/model.tsx

```diff
@@ -701,19 +689,19 @@ function SubmitJob(instanceId: any, user: any, job: any, callback: any, failure:
 }
 
 
-function JobInfo(clusterName: any, instanceId: any, user: any, jobId: any, callback: any, failure: any) {
+function JobInfo(_clusterName: any, instanceId: any, user: any, jobId: any, successCallback?: Callback, failureCallback?: Callback) {
```

Comment:
question: why is `clusterName` here if it's not used?

Reply:
fixed.